rosidl_typesupport_opensplice_c/resource/srv__type_support_c.cpp.em

```diff
@@ -29,8 +29,6 @@
 #include "@(spec.pkg_name)/srv/dds_opensplice/ccpp_Sample_@(spec.srv_name)_Response_.h"
 #include "rosidl_typesupport_opensplice_c/identifier.h"
 
-#include "@(spec.pkg_name)/srv/dds_opensplice_c/@(get_header_filename_from_msg_name(spec.srv_name))__request__type_support.h"
-#include "@(spec.pkg_name)/srv/dds_opensplice_c/@(get_header_filename_from_msg_name(spec.srv_name))__response__type_support.h"
 #include "@(spec.pkg_name)/srv/dds_opensplice/@(get_header_filename_from_msg_name(spec.srv_name))__type_support.cpp"
```

Comment:
@wjwwood testing this locally, I get a segfault in the tests I have for C parameters. I'm not entirely sure, but I think this needs to be changed to "__type_support_c.cpp". Will do more testing and push a fix if I find one.